ERCS/erc-XXXX.md

```diff
@@ -13,60 +13,64 @@ requires: 712, 1193
-* the user is victim of social engineering
+- Editing the calldata in order to siphon funds or otherwise change the transaction outcome
+- Modify the parameters of an EIP-712 request
+- Obtain a replayable signature from the wallet
+- TODO
 
-The effect can be pernicious as end users will not necessarily realise there is no chain of trust between the dApp backend and the wallet that will send a transaction on the user's behalf: a MITM attack can
-capitalize on trusted dApp to syphon funds.
+Even if the user realizes that requests from the dapp may be tampered with, they have little to no recourse to mitigate the problem.
+Overall, the lack of a chain of trust between the dapp and the wallet hurts the ecosystem as a whole:
 
-Such an attack can be used in a variety of ways:
-
-* Modify the call data on the fly, as a user is transacting
-* Obtain a replayable signature from the user's wallet
-* TODO
-
-Overall, the lack of a chain of trust between the dApp and the wallet hurts the ecosystem as a whole:
-
-* users cannot rely on trusting well known dApp, and are at risk of losing funds if they do not manually verify transactions before signing
-* dApp maintainer are at risk of losing their trusted reputation if an attacker finds a viable MITM attack
+- Users cannot simply trust otherwise honest dapps, and are at risk of losing funds
+- Dapp maintainers are at risk of hurting their reputations if an attacker finds a viable MITM attack
 
 ## Specification
 
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
 ### Overview
 
-We are proposing to use the domain certificate of a dApp as a root of trust to establish a trust chain as follow:
+We propose to use the dapp's domain certificate of a root of trust to establish a trust chain as follow:
 
-1. end user's browser verifies the domain certificate and display appropriate warning if overtaken
-2. the DNS record of the dApp hosts a TXT field pointing to a URL where a JSON manifest is hosted (this file SHOULD be at a well known address such as <https://domain.com/.well-known/twit.json>)
-3. the config file contains an array of tuple `[alg, public key]`
-4. for secure interaction, a dApp would first call their backend to securely sign the payload with a private key
-5. the encrypted payload would be sent to the wallet via the `secureRequest` JSON api call
-6. the wallet would verify the signature before sending the payload to the `request` JSON api call
+1. The user's browser verifies the domain certificate and displays appropriate warnings if overtaken
+2. The DNS record of the dapp hosts a TXT field pointing to a URL where a JSON manifest is hosted
+   - This file SHOULD be at a well known address such as <https://domain.com/.well-known/twit.json>
```

Comment:
Is `.well-known` what we want here, or is this just a placeholder?

Also, should this be upgraded to a MUST?

Reply:
.well-known is standard for identity stuff, we should keep it unless the community complains